src/zabapgit_dot_abapgit.prog.abap

```diff
@@ -103,7 +108,7 @@ CLASS lcl_dot_abapgit IMPLEMENTATION.
     DATA: ls_data TYPE ty_dot_abapgit.
 
 
-    ls_data-master_language = iv_master_language.
+    ls_data-master_language = sy-langu.
```

Comment:
Although this code is just moved ... I think it is bit straight forward. What if master language is actually different ? Maybe it would be better to select it from some object in package ? (maybe from package itself from tadir ?)